project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+class PreferenceForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cuisine: '',
+      distance: '',
+      dining_experience: '',
+      price_level: '',
+      latitude: '',
+      longitude: '',
+      open: true,
+    };
+    this.changeState = this.changeState.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  // Updates the state of the input element so it holds the chosen value.
+  changeState(event) {
+    const value =
+      event.target.name === 'open' ? event.target.checked : event.target.value;
+    this.setState({ [event.target.name]: value });
+  }
+
+  handleSubmit(event) {
+    alert('the cuisine you chose is: ' + this.state.cuisine);
```

Comment:
is this a pop-up alert? I wonder if this'll look too intrusive (and if we need an alert like this at all?)

Reply:
Jesse told me to just add this for now as a filler(until we do more with the handleSubmit function)

Reply:
👍  can you add a TODO here to address this in follow ups then?